pex/BUILD

```diff
@@ -22,7 +22,7 @@ genrule(
         # Workaround really long shebang lines breaking on linux:
         # Use a /tmp path, but keep the actual venv inside the bazel outdir.
         # Avoids having to worry about cleanup, even if sandboxing is off.
-        TMPF=$$(mktemp)
+        TMPF=$$(mktemp 2>/dev/null || mktemp -t tmp)
```

Comment:
Wow, I wonder how this ever worked on MacOS - did mktemp's behaviour change in a recent macos release?